walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -71,23 +71,23 @@ The Fuse Aggregation App provides information about flights. You deploy it using
 
 . Select the option to *Build and Deploy Online* in the *Target Environment* step.
 
-. Select the *Authorize* button to allow Launcher create resources in the OpenShift cluster. This triggers an OAuth flow, and eventually redirects back to the {launcher-service} wizard.
+. Click the *Authorize* button to allow Launcher create resources in the OpenShift cluster. This triggers an OAuth flow, and eventually redirects back to the {launcher-service} wizard.
 
-. Select the blue down-arrow icon to continue the wizard.
+. Click the blue down-arrow icon to continue the wizard.
 
 . Select the *Aggregation* mission and *Fuse* runtime in the *Mission & Runtime* step
 
-. Select the blue down-arrow icon to continue the wizard.
+. Click the blue down-arrow icon to continue the wizard.
 
 . Authorize your Github Account
-.. Select the *Log In & Authorize Account* button in the *Authorize Git Provider* step.
+.. Click the *Log In & Authorize Account* button in the *Authorize Git Provider* step.
 .. This opens a Github OAuth page.
-.. Select the *Authorize <organisation/user>* button at the bottom.
+.. Click the *Authorize <organisation/user>* button at the bottom.
 .. You will be redirected back to the {launcher-service} wizard.
 
-. Select all blue down-arrow icons to continue the wizard, accepting the git repository name.
+. Click all the blue down-arrow icons to continue the wizard, accepting the git repository name.
```

Comment:
I would remove "the wizard".

Also, " accepting the git repository name." seems slightly odd (but this is due to me not having context).